### PR TITLE
[json] Version 3.11.3; build with cmake

### PR DESCRIPTION
**Pull request by smuzaffar**
Building with cmake allows other cmake based projects to find our installation

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

File: json.spec [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+### RPM external json 3.11.3
 ## NOCOMPILER
 
-Source: https://github.com/nlohmann/%{n}/releases/download/v%{realversion}/include.zip
+Source: https://github.com/nlohmann/json/archive/refs/tags/v%{realversion}.tar.gz
 
 %prep
-%setup -c
+%setup -n %{n}-%{realversion}
+sed -i -e 's|JSON_BuildTests_INIT ON|JSON_BuildTests_INIT OFF|' CMakeLists.txt
```
Comment (sethrj) [inline json.spec]: I think you can just add `-DJSON_BuildTests=OFF` to the cmake line for the same effect (but more robust against future changes)

Comment (smuzaffar) [reply json.spec]: thanks @sethrj

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**